examples/plot_season_summary.py

```diff
@@ -0,0 +1,168 @@
+"""
+Season Summary Visualization
+==================================
+This example demonstrates how to make an interactive season summarization
+dashboard showing points scored by each driver at each round.
+"""
+
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.io import show
+from plotly.subplots import make_subplots
+
+import fastf1 as ff1
+
+
+##############################################################################
+# Fetch the event schedule. Skip the first row, which is the pre-season test.
+season = 2024
+schedule = ff1.get_event_schedule(season).iloc[1:]
+
+##############################################################################
+# Get each driver's finishing positions and points total for each round.
+standings = []
+# Shorten the event names by trimming Grand Prix from the name.
+# This will be used to label our graph.
+short_event_names = []
+
+for _, event in schedule.iterrows():
+    event_name, round_number = event["EventName"], event["RoundNumber"]
+    short_event_names.append(event_name.replace("Grand Prix", "").strip())
+
+    # Only need to load the results data
+    race = ff1.get_session(season, event_name, "R")
+    race.load(laps=False, telemetry=False, weather=False, messages=False)
+
+    # Add sprint race points if applicable
+    # For 2021 and 2022 season, this is called 'sprint'
+    # For 2023, this is called 'sprint_shootout'
```

Comment:
I was confused by what this refers to until I figured out that this is about the comparison for the event format if someone wanted to use this for another season. This could maybe be worded a bit more clearly.